### ISSUE TITLE
Adding all available values should maintain order

### ISSUE THREAD

**Issue by allanaaa** [omeka-s-modules/FacetedBrowse#36]
I'd like to see facets ordered from most to least frequency, when using the "Add all" button. Currently it seems to order them by some internal ID or creation date, which is counter-intuitive to the interface. 

<img width="402" alt="Screenshot 2023-11-15 at 12 40 19" src="https://github.com/omeka-s-modules/FacetedBrowse/assets/28012148/ddefc5fd-4a29-4d35-81e9-883f1f3216f9">

<img width="403" alt="Screenshot 2023-11-15 at 12 34 23" src="https://github.com/omeka-s-modules/FacetedBrowse/assets/28012148/3adfc678-8e75-4e3e-88c9-5b140771faff">

**Comment by jimsafley** [comment]
I agree that clicking the "Add all" button should maintain order. Unfortunately, the use of a multi-select (modified by [Chosen](https://harvesthq.github.io/chosen/)) makes that impossible because it always maintains the original order of the select options, regardless of how it's populated. It may be possible to change how we populate the select, but that would require a non-negligible change to the core; and it may not be advisable anyway because the query that fetches item count is intensive and could reach some memory or time limit. We get away with it here because "Show all available..." is optional.

The same goes for #38: there's no way to rearrange select options on a multi-select. Chosen improves UI and adds some features but it doesn't allow rearranging. I think the way forward here is to explore alternatives to the native multi-select. I haven't looked into it much, but possibilities are [Selectize](https://github.com/selectize/selectize.js) and [Choices](https://github.com/Choices-js/Choices). Of course we could develop a custom solution that circumvents the use of a multi-select, but that's considerably more ambitious and probably not worth our time.
